api/src/test/java/org/openmrs/module/idgen/service/IdentifierSourceServiceTest.java

```diff
@@ -104,12 +104,25 @@ public void getIdentifierSource_shouldReturnASavedRestIdentifierGenerator() thro
 	@Test
 	@Verifies(value = "should return a saved identifier pool", method = "getIdentifierSource(Integer)")
 	public void getIdentifierSource_shouldReturnASavedIdentifierPool() throws Exception {
-		IdentifierPool idpool = (IdentifierPool)iss.getIdentifierSource(3);
+		IdentifierPool idpool = (IdentifierPool)identifierSourceService.getIdentifierSource(3);
 		Assert.assertEquals(idpool.getName(), "Test Identifier Pool");
 		Assert.assertEquals(idpool.getBatchSize(), 1000);
 		Assert.assertEquals(5, idpool.getAvailableIdentifiers().size());
 		Assert.assertEquals(idpool.getUsedIdentifiers().size(), 0);
 	}
+	
+	/**
+	 * @see {@link IdentifierSourceService#retireIdentifierSource(IdentifierSource, String)}
+	 * 
+	 */
+	@Test
+	@Verifies(value = "should retire an IdentifierSource from the system", method = "retireIdentifierSource(IdentifierSource, String)")
+	public void retireIdentifierSource_shouldRetireAnIdentifierSourceFromTheSystem() throws Exception {
+		IdentifierSource identifierSource = identifierSourceService.getIdentifierSource(1);

```

Comment:
Can you avoid repeating 1 in several places?

Reply:
fixed